### ISSUE TITLE
Commaf which rounds to X decimal places

### ISSUE THREAD

**Issue by bramp** [dustin/go-humanize#18]
``` go
humanize.Commaf(10772.08207) // Returns "10,772.082074"
```

Could I have a

``` go
humanize.Commaf(10772.08207, 2) // Returns "10,772.08"
```

?

Or do you suggest an alternative way to round the number, before humanize'ng? Thanks.

**Comment by kristen1980** [comment]
I'd like to second this. Right now I am doing:

      fl := 123456.789  // this works as expected and gives "123,456.79"
      fl := 123456.0  // this gives "123,456"....notice it truncates the decimals if they are .00000000
      sss := fmt.Sprintf("%.2f", fl)
      if ffl, err := strconv.ParseFloat(sss, 64); err == nil {  // convert it back to a float...very clunky!
        log.Println(sss, humanize.Commaf(ffl))
      } else {
        log.Println(err)
      }
However, as you can see, the decimals get truncated if they are zero....this also seems very clunky.

**Comment by shurcooL** [comment]
Was this issue resolved by PR #60 (which has been recently merged), or is there more to do here? /cc @bramp

**Comment by bramp** [comment]
Yup #60 fixes this issue
